modules/exploits/linux/http/zimbra_xxe_rce.rb

```diff
@@ -0,0 +1,254 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::Remote::HttpServer
+  include Msf::Exploit::FileDropper
+  include REXML
```

Comment:
Interesting that you used this as a mixin. It's not usually used that way. Are you using any special methods that need including?

Reply:
Building the XML document

Reply:
Those methods look like they're acting on the object, not the class. Why not `REXML::Document.new` to instantiate? What methods need to be mixed in?